hw/bsp/imxrt/family.c

```diff
@@ -127,19 +135,397 @@ void board_init(void)
   uart_config.enableTx = true;
   uart_config.enableRx = true;
 
-  if ( kStatus_Success != LPUART_Init(UART_PORT, &uart_config, UART_CLK_ROOT) ) {
+  if (kStatus_Success != LPUART_Init(UART_PORT, &uart_config, UART_CLK_ROOT)) {
     // failed to init uart, probably baudrate is not supported
     // TU_BREAKPOINT();
   }
 
   //------------- USB -------------//
   // Note: RT105x RT106x and later have dual USB controllers.
-  init_usb_phy(0); // USB0
+  init_usb_phy(0);// USB0
 #ifdef USBPHY2
-  init_usb_phy(1); // USB1
+  init_usb_phy(1);// USB1
 #endif
 }
 
+/* MPU configuration. */
+#if __CORTEX_M == 7
+static void BOARD_ConfigMPU(void) {
```

Comment:
@HiFiPhile is this BOARD_ConfigMPU() disable cache memory for M7. I actually prefer to have cache enabled for imxrt by default (CFG_TUD/H_MEM_DCACHE_ENABLE=1). Since that is what most user with this chip would do. Also having it enable in example will make sure we test with cacheable memory.

Reply:
`BOARD_ConfigMPU()` is copied from ksdk, which enables DCache and ICache.
If you want to test examples on cached OCRAM you can simply modify the linker, default linker script from ksdk use DTCM.

Reply:
thanks @HiFiPhile for the info, that sound good. I am currrently messing around with mcux config tool for mpu(), these are too complicated 
![image](https://github.com/user-attachments/assets/c84cff64-a27a-4b8f-82b8-3de97323fbf7)